dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/AlertDaoTest.java

```diff
@@ -23,8 +23,8 @@
 
 import java.util.List;
 
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 import org.junit.runner.RunWith;
```

Comment:
We need to remove this, use `ExtendWith` instead as stated in #12301 SOP.

Reply:
done